### DISCUSSION TITLE
How to Embed a standard page under / (root)

### DISCUSSION THREAD

**Discussion by FredWagner-sys** [btcpayserver/btcpayserver#2547] [Technical Support]
I hope someone can help me. I want to run BTC at BtcPay.domain.xx. But a normal (CMS) page should be accessible on the main domain.xx. How do I do that. In the BtcPayServer as a Docker Install?

**Comment by woutersamaey** [comment]
Since you are running btcpay server on a subdomain, and want a webpage on the root domain, this has nothing to do with btcpay server.
Its a different domain, so you should put the page there like you would normally do without btcpay.
This involves placing an html file on you webserver.
